cardano-db-sync/src/Cardano/DbSync.hs

```diff
@@ -236,52 +234,68 @@ runDbSyncNodeNodeClient iomgr trce plugin nodeConfig (SocketPath socketPath) = d
                           (TraceSendRecv (Handshake NodeToClientVersion CBOR.Term)))
     handshakeTracer = toLogObject $ appendName "Handshake" trce
 
-
-localInitiatorNetworkApplication
-  :: forall blk.
-     (blk ~ ByronBlock)
-  -- TODO: the need of a 'Proxy' is an evidence that blk type is not really
-  -- needed here.  The wallet client should use some concrete type of block
-  -- from 'cardano-chain'.  This should remove the dependency of this module
-  -- from 'ouroboros-consensus'.
-  => Trace IO Text
+    dbSyncVersions txv = foldMapVersions
```

Comment:
Could you add the type signature, it's not needed, but it makes it easier at imes.

Reply:
done